test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -56,6 +56,24 @@ proc ::AlgorandGoal::Abort { ERROR } {
     exit 1
 }
 
+# Utility method to test the process returned value
```

Comment:
nit: please add a comment:
method returns 1 in case of a normal or abnormal process error or zero otherwise.